containers-sig-laws/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```

Comment:
I think that this `Setup.hs` file is not needed and can be removed. `cabal` works okay without this file